tree/20_cfe_basics/dry_run_lib.cf

```diff
@@ -56,77 +56,86 @@ bundle agent set_dry_run_mode(mode)
 #  - ... and all promise types that are not for the agent
 ###############################################################################
 
+body file control
+{
+   namespace => "bodydefault"; 
```

Comment:
No, this is the namespace used in CFEngine for default bodies (https://docs.cfengine.com/latest/reference-language-concepts-bodies.html#default-bodies)